packages/api-core/src/me/types.ts

```diff
@@ -6,28 +6,81 @@ export interface SocialLoginConnection {
 	service_user_id: string;
 }
 
+export interface UserMetaData {
+	links: Record< 'self' | 'help' | 'site' | 'flags', string >;
+	data: {
+		flags: {
+			active_flags: string[];
+		};
+	};
+}
+
 export interface User {
 	ID: number;
-	username: string;
-	display_name: string;
+	abtests: Record< string, string >;
+	atomic_site_count: number;
+	atomic_visible_site_count: number;
 	avatar_URL?: string;
-	language: string;
-	locale_variant: string;
-	logout_URL?: string;
+	date: string;
+	display_name: string;
 	email: string;
 	email_verified: boolean;
-	is_valid_google_apps_country: boolean;
+	had_hosting_trial: boolean;
+	has_jetpack_partner_access?: boolean;
 	has_unseen_notes: boolean;
+	i18n_empathy_mode: boolean;
+	is_google_domain_owner?: boolean;
+	is_subscription_only: boolean;
+	is_valid_google_apps_country: boolean;
+	jetpack_partner_types?: string[];
+	jetpack_site_count?: number;
+	jetpack_visible_site_count?: number;
+	language: string;
+	lasagna_jwt: string;
```

Comment:
I think I agree with @StevenDufresne; the type is becoming confusing.

Is it possible to only add the legacy bootstrapped user fields in this v2 type, and not the other things that v2 has never used like these?

Reply:
Ohh.... I think I understand now.... thanks for explaining... 😅